Process/line_warpers.py

```diff
@@ -0,0 +1,92 @@
+"""Module for and generating altering line segments to assist processing and defining puzzle pieces."""
+
+import numpy as np
+import math
+
+def rotate(origin, point, angle):
```

Comment:
I would name this something more descriptive, like `rotate_point_around_reference` (or similar)